### ISSUE TITLE
Solution is not flexible?

### ISSUE THREAD

**Issue by Duplicated** [freeCodeCamp/freeCodeCamp#15567]
Challenge [adjust-the-hover-state-of-an-anchor-tag](http://beta.freecodecamp.com/en/challenges/applied-visual-design/adjust-the-hover-state-of-an-anchor-tag) has an issue.
User Agent is: <code>Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/59.0.3071.115 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

```html


<style>
  a {
    color: #000;
  }
  
  a:hover {
    color: blue;
  }
  
</style>
<a href="http://freecatphotoapp.com/" target="_blank">CatPhotoApp</a>

```
Earlier, I tried `color: #00f` for my `a:hover` pseudo-class, but apparently it wasn't accepted as a solution, even though it's literally the *exact same* shade of blue. What gives?

**Comment by erictleung** [comment]
@Duplicated thanks for the issue. I think this is *might* be a duplicate of this issue https://github.com/freeCodeCamp/freeCodeCamp/issues/10595. 

cc/ @freeCodeCamp/moderators right now, this challenge uses a regex to check the color in the hover tag. A similar fix was implemented in https://github.com/freeCodeCamp/freeCodeCamp/pull/13389 and would be what I suggest to fix this, but I'm not sure if there is a better solution.

**Comment by Duplicated** [comment]
> A similar fix was implemented in #13389
So basically you'd need `assert()` to account for every possible answer? I mean, these would all be technically correct:
```css
a:hover {
    color: blue;
}

a:hover {
    color: #0000ff;
}

a:hover {
    color: #00f;
}

a:hover {
    color: rgb(0, 0, 255);
}

a:hover {
    color: rgba(0, 0, 255, 1.0);
}

a:hover {
    color: hsl(240, 100%, 50%);
}
```
right? Can Regex conditionals account for all these cases?

By the way, these were mentioned in the Applied Visual Design chapter, so I assume any of them should be fair game.

**Comment by Greenheart** [comment]
To avoid checking for all valid color values, maybe we could compare the computed color by getting the actual anchor element and comparing it's styles with the expected color? If we know that the browser stores computed colors in format Y for example, we only have to do one comparison.

I'm not sure if it's possible to find the computed css for hover states though. I assume the `<DOMElement>.styles` object only contains what's currently being displayed in the browser.
